test/benchmarks/kubernetes_state/main.go

```diff
@@ -231,6 +230,10 @@ func main() {
 	pprof.StopCPUProfile()
 
 	cancel()
-
 	fmt.Printf("KSMCheck.Run() returned %v in %s\n", err, elapsed)
+
+	if err = file.Close(); err != nil {
+		fmt.Println("failed to close \"cpuprofile.pprof\": %v\n", err)
```

Comment:
We should use the `log` module instead of `fmt` to get the logs formatted as expected:

```suggestion
		log.Println("failed to close \"cpuprofile.pprof\": %v\n", err)
```